sandboxie-plus/Sandboxie
# Better icons for (dis-)connect to show enabled/disabled states.

Feature:
* Provide coloured connect/disconnect icons so enabled and disabled states are easier to distinguish

QA Suggestions:
* Test on platforms other than Win10x64
* Ensure enabled disabled states work
* Ensure dark mode works


![image](https://github.com/sandboxie-plus/Sandboxie/assets/28550406/c2dcc643-38fc-44df-ae9f-bc9a228cf91f)
